EvidenceApi/V1/Controllers/DocumentSubmissionsController.cs

```diff
@@ -58,6 +61,32 @@ public async Task<IActionResult> UpdateDocumentSubmissionState([FromRoute][Requi
             }
         }
 
+        /// <summary>
+        /// Updates the visibility of a document submission
+        /// </summary>
+        /// <response code="200">Updated</response>
+        /// <response code="400">Request contains invalid parameters</response>
+        /// <response code="404">Document submission cannot be found</response>
```

Comment:
Currently this means submission not found or already hidden.

Reply:
Now it should be fine (new gateway called in the usecase to return all submissions)